<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="add_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>111</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить вершину</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="vert_insert">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="vertex1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="vertex2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="add_edge">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>40</y>
      <width>91</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить грань</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="weight">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="vert_text2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Вершина 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="weight_text">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>110</y>
      <width>37</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>Вес</string>
    </property>
   </widget>
   <widget class="QLabel" name="vert_text1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Вершина 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="seller_button">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>70</y>
      <width>91</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Коммивояжер</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_vertex">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>110</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить вершину</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_edge">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>160</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить грань</string>
    </property>
   </widget>
   <widget class="QPushButton" name="change_edge">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>160</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Изменить грань</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
